application/modules/front/views/collection/printhadith.php

```diff
@@ -28,8 +28,10 @@
 				else {
 					echo "$collection->englishTitle $hadithNumber";
 				}
-				echo "</div>";
 			}
+			else
+				echo "$collection->englishTitle $hadithNumber";
+			echo "&nbsp;</div>";
```

Comment:
We shouldn't have this. There should be no sticky when the book status is not >= 4.